exonum-java-binding-common/src/main/java/com/exonum/binding/common/serialization/StandardSerializers.java

```diff
@@ -153,7 +153,7 @@
    * Returns a serializer of ZonedDateTime instances.
    */
   public static Serializer<ZonedDateTime> zonedDateTime() {
```

Comment:
Why is this public even though this class is suppossed to be internal (especially with no documentation of its limitations)?